### PR TITLE
setup.py generator

### PR DESCRIPTION
**Pull request by shubham-shay-te**

### CODE REVIEW

File: core_lib_generator/template_core_lib/template_core_lib/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Version of the core_lib
+__version__ = "0.0.0.9"
```
Comment (shay-te) [inline core_lib_generator/template_core_lib/template_core_lib/__init__.py:2]: what is this specific version? 
how are we replacing it?

**Review by shay-te — CHANGES_REQUESTED** [review]

File: core_lib/helpers/shell_utils.py [hunk 2]
```diff
@@ -86,6 +88,35 @@ def input_list(list_value: list, title: str, default_value: int = None):
...
+            result = user_input
+    return result
+
+
+def input_url(title: str, default_value: str = None) -> str:
+    result = None
+    while result is None:
+        user_input = _input(f'{title} {_print_default(default_value)}: ')
+        user_input = _get_value(user_input, default_value)
+        regex = re.compile(
```
[the remark this answers is not in the document]
Comment (shubham-shay-te) [reply core_lib/helpers/shell_utils.py:106]: this was my first implementation, what happens is it only checks for `://` in the URL and returns true so if we have `random://google.com` then too it is going to pass, this regex implementation is of Django's URL validations

File: core_lib_generator/config_collectors/setup_collector.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3.7',
+        'Topic :: Software Development',
+        'Topic :: Software Development :: Libraries',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Typing :: Typed',
+    ]
+    user_classifiers = []
+    while add_classifiers:
+        user_classifiers_input = input_list(classifiers, 'Select the classifiers for your project')
```
[the remark this answers is not in the document]
Comment (shay-te) [reply core_lib_generator/config_collectors/setup_collector.py:39]: for input_string do 
input_str(........, validate_value: callback: Callable[[dict], Awaitable[dict]] = None, title_validate_value_fail: str = 'Value is not valid ....')

File: core_lib/helpers/shell_utils.py [hunk 4]
```diff
@@ -105,11 +106,36 @@ def input_list(
         user_input = _get_value(user_input, default_value)
         if is_int(user_input) and len(list_value) >= int(user_input) > 0:
             if list_value[int(user_input) - 1] in existing_selected:
-                return input_list(list_value, title_existing_selected, default_value, existing_selected)
+                return input_list(
```
Comment (shay-te) [inline core_lib/helpers/shell_utils.py:109]: no recursive please

File: core_lib_generator/template_core_lib/setup.py [hunk 5]
```diff
@@ -2,14 +2,19 @@
...
+# template_core_lib_import
 
 from setuptools import find_namespace_packages, setup
 from pip._internal.network.session import PipSession
 from pip._internal.req import parse_requirements
 
 dir_path = os.path.dirname(os.path.realpath(__file__))
 install_reqs = parse_requirements(os.path.join(dir_path, 'requirements.txt'), session=PipSession)
+requirements = []
+try:
```
Comment (shay-te) [inline core_lib_generator/template_core_lib/setup.py:14]: is this if statment needed now?

**Review by shay-te — CHANGES_REQUESTED** [review]

File: core_lib_generator/template_core_lib/setup.py (hunk 5 above)
Comment (shubham-shay-te) [reply core_lib_generator/template_core_lib/setup.py:14]: it's for generating the list of requirements which was failing earlier

**Review by shay-te — APPROVED** [review]

**Pull request merged**